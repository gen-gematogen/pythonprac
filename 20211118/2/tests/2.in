class C:
        num = Num()

c, d = C(), C()
c.num = 2
d.num = 10
print(c.num)
print(d.num)
c.num = "password)"
print(c.num+d.num)
d.num = ""
print(c.num+d.num)
