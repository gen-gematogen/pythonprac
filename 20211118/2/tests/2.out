2
10
19
9
